src/tests/Microsoft.PowerFx.Core.Tests/ExpressionTestCases/First.txt

```diff
@@ -89,3 +89,8 @@ Table({Value:5})
 >> FirstN(If(false, Table({a:1}), Table({b:2})), 1)
 Table({})
 
+>> First(Table(If(false, {x:1, y:1}, {x:1, z:2}), {x:9,z:9})).z
```

Comment:
Can you add a comment to help explain why. 
Actually, it'd even be fine to include this as a test case right before to help explain it. 

If(false, {x:1, y:1}, {x:1, z:2})

Reply:
added